symfony/client/src/orangehrmCorePlugin/pages/updatePassword/UpdatePassword.vue

```diff
@@ -0,0 +1,148 @@
+<!--
```

Comment:
@VageeshOrangeHRM can you move all pages to `OrangehrmPimPlugin`

Reply:
done